mvpa2/misc/neighborhood.py

```diff
@@ -296,7 +299,7 @@ def _get_increments(self, ndim):
 
         if not len(res):
             warning("%s defines no neighbors" % self)
-        return res
+        return np.vstack([np.zeros(ndim,dtype='int'),res]) if self.include_center else res
```

Comment:
you never assign it to self.  So it leads to tests failures:  https://travis-ci.org/PyMVPA/PyMVPA/jobs/47877969  and for proper this PR should have a test assigned (just set this parameter and verify that you do get center reported, and that it is first as you promised)